### ISSUE TITLE
Broken ivi support

### ISSUE THREAD

**Issue by deathnoragami** [yt-dlp/yt-dlp#11069]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Rus

### Provide a description that is worded well enough to be understood

Cookies don't work, downloads video without subscription once in a while. video with subscription doesn't download at all.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '--cookies', 'ivi.ru_cookies.txt', 'https://www.ivi.ru/watch/vasha-chest-2024/538722']
[debug] Encodings: locale cp1251, fs utf-8, pref cp1251, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.08.06 from yt-dlp/yt-dlp [4d9231208] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg N-117043-g8707c8660d-20240915 (setts), ffprobe N-117043-g8707c8660d-20240915
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.07.04, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.35.5, urllib3-2.2.2, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1830 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: stable@2024.08.06 from yt-dlp/yt-dlp
yt-dlp is up to date (stable@2024.08.06 from yt-dlp/yt-dlp)
[ivi] Extracting URL: https://www.ivi.ru/watch/vasha-chest-2024/538722
[ivi] 538722: Downloading timestamp JSON
[ivi] 538722: Downloading video JSON
[ivi] 538722: Downloading video JSON
ERROR: [ivi] 538722: Unable to download video 538722: Не смогли определить версию по переданным site=s183 и app_version=None
  File "yt_dlp\extractor\common.py", line 740, in extract
  File "yt_dlp\extractor\ivi.py", line 133, in _real_extract
```

**Comment by deathnoragami** [comment]
![image](https://github.com/user-attachments/assets/19577162-8f64-4417-a4ff-73e3ecf96221)
I don't think it's possible to talk about DRM, because IDM quietly takes all subtitle data, language changes

**Comment by bashonly** [comment]
Possibly related #7021

**Comment by deathnoragami** [comment]
> Possibly related #7021

no, various errors, there the error was only that the series was old and had a different format video 4:3. And yt does not have inside finding such formats, which is why it had such an error. I was able to download that series by writing directly what exactly to download.
